Description: Strip 'uwsgi' shebang line from uWSGI examples
 uWSGI examples are placed to /usr/share/doc/uwsgi-core/examples, so
 Lintian warns about relative interpreter path with informational tag
 (I: example-interpreter-not-absolute).
 .
 As user can install only the uwsgi-core package (without uwsgi-plugin-python
 package), we have no guaranteed path to uWSGI that could be placed into
 shebang line. So it's stripped.
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-07-26

--- a/ugreenchat.py
+++ b/ugreenchat.py
@@ -1,4 +1,3 @@
-#!./uwsgi
 import uwsgi
 
 from bottle import route, default_app, request, response, redirect
--- a/decoratortest.py
+++ b/decoratortest.py
@@ -1,4 +1,5 @@
-#!./uwsgi --import decoratortest -s :3035 -M --spooler myspool --enable-threads -p 8 --spooler-ordered
+# uWSGI options suggested for this example tryout:
+# --master --enable-threads --spooler /tmp/uwsgispool --spooler-ordered
 import uwsgi
 
 from uwsgidecorators import *
